lib/cmr/cmr_client.rb

```diff
@@ -42,6 +42,16 @@ def get_collections_by_post(query, token = nil)
       post(url, query.to_query, headers.merge(token_header(token)))
     end
 
+    def get_variables(options = {}, token = nil)
```

Comment:
actually I was trying to mirror `get_collections` above, though they are similar and it could be either. I had tested trying to make a `get_variables_by_post`, but they only have the GET endpoint. Should I change the name of the method or leave it?